package.json

```diff
@@ -7,20 +7,46 @@
     "url": "https://github.com/withastro/astro.git"
   },
   "scripts": {
-    "release": "yarn build:all && changeset publish",
-    "benchmark": "yarn workspace astro run benchmark",
-    "build": "yarn build:core",
-    "build:one": "lerna run build --scope",
-    "build:all": "lerna run build --scope \"{astro,create-astro,@astrojs/*}\"",
-    "build:core": "lerna run build --scope \"{astro,@astrojs/parser,@astrojs/markdown-remark}\"",
-    "dev": "yarn dev:core --parallel --stream",
-    "dev:one": "lerna run dev --scope --parallel --stream",
-    "dev:all": "lerna run dev --scope \"{astro,create-astro,@astrojs/*}\" --parallel --stream",
-    "dev:core": "lerna run dev --scope \"{astro,@astrojs/parser,@astrojs/markdown-remark}\" --parallel --stream",
-    "format": "prettier -w .",
+    "release": "yarn build && changeset publish",
+    "build": "turbo run build --no-deps --scope=astro --scope=create-astro --scope=\"@astrojs/*\"",
+    "dev": "turbo run dev --no-deps --no-cache --parallel --scope=astro --scope=create-astro --scope=\"@astrojs/*\"",
+    "test": "turbo run test --scope=astro",
+    "test:templates": "turbo run test --scope=create-astro",
+    "benchmark": "turbo run benchmark --scope=astro",
     "lint": "eslint \"packages/**/*.ts\"",
-    "test": "yarn workspace astro run test",
-    "test:templates": "lerna run test --scope create-astro --stream"
+    "format": "prettier -w ."
+  },
+  "turbo": {
```

Comment:
is there a `turbo.json` or something like that? It would be nice not to impose this configuration on end-users if we can avoid it.

Reply:
It does not appear so! This is only in the workspace root `package.json` file, though, so it will never ship to end-users.

Reply:
ah, right, cool then